### DISCUSSION TITLE
code action view

### DISCUSSION THREAD

**Discussion by yriveiro** [folke/noice.nvim#221] [Ideas]
Hi,

Any chance to override `textDocument/codeAction` menu as it does with `hover` and `signatureHelp`?

<img width="646" alt="Screenshot 2022-11-13 at 22 30 40" src="https://user-images.githubusercontent.com/934763/201547900-4d9f2973-40b5-4f44-b918-b0dd2e8a3904.png">

**Comment by folke** [comment]
`codeAction` uses `vim.ui.select` under the hood, so you could use https://github.com/stevearc/dressing.nvim for example for a better ui.

**Reply by yriveiro** [reply]
Seems to be a nice plugin, I will give it a try, thanks @folke
